force-app/main/default/lwc/geGatewaySelectWidget/geGatewaySelectWidget.js

```diff
@@ -155,6 +155,20 @@ export default class GeGatewaySelectWidget extends LightningElement {
             this.gatewayOptions.push({label: gateway.gatewayName, value: gateway.id});
         }
         this.gatewayOptions = this.gatewayOptions.sort((a, b) => a.label >= b.label ? 1 : -1);
+
+        if (this._widgetMode !== GATEWAY_MANAGEMENT_MODE) {
```

Comment:
Implementing the helper I mentioned could work here as well `!onSettingsPage`

Reply:
@lparrott Implemented your suggestion, but went with different names since both the management page and the template page seem sorta like settings pages.